extensions-core/druid-pac4j/src/main/java/org/apache/druid/security/pac4j/OIDCConfig.java

```diff
@@ -35,16 +36,21 @@
   @JsonProperty
   private final String discoveryURI;
 
+  @JsonProperty
+  private final String oidcClaim;
+
   @JsonCreator
   public OIDCConfig(
       @JsonProperty("clientID") String clientID,
       @JsonProperty("clientSecret") PasswordProvider clientSecret,
-      @JsonProperty("discoveryURI") String discoveryURI
+      @JsonProperty("discoveryURI") String discoveryURI,
+      @JsonProperty("oidcClaim") String oidcClaim
```

Comment:
Since this parameter is used to specify an arbitrary claim value as the Druid AuthenticationResult's identity, perhaps we can give it a name such as `identityClaim`